configure.ac

```diff
@@ -274,12 +274,25 @@ AC_ARG_ENABLE([suid],
 
 AS_IF([test "x$enable_suid" != "xno"], [
     AC_MSG_RESULT([yes])
-    BUILD_SUID="action-suid create-suid copy-suid expand-suid export-suid import-suid mount-suid"
+    BUILD_SUID="action-suid copy-suid expand-suid export-suid import-suid mount-suid"
```

Comment:
You can check by looking at the build on Travis: it's uses an old version of mkfs.ext3 so it needs SUID rights to work.